reactor-netty-http/src/main/java/reactor/netty/http/server/AbstractHttpServerMetricsHandler.java

```diff
@@ -208,6 +212,9 @@ private long extractProcessedDataFromBuffer(Object msg) {
 		else if (msg instanceof Buffer buffer) {
 			return buffer.readableBytes();
 		}
+		else if (msg instanceof HttpContent httpContent) {
```

Comment:
```
/.../reactor-netty/reactor-netty-http/src/main/java/reactor/netty/http/server/AbstractHttpServerMetricsHandler.java:215: warning: [rawtypes] found raw type: HttpContent
                else if (msg instanceof HttpContent httpContent) {
                                        ^
  missing type arguments for generic class HttpContent<R>
  where R is a type-variable:
    R extends HttpContent<R> declared in interface HttpContent
```

```suggestion
		else if (msg instanceof HttpContent<?> httpContent) {
```

Reply:
applied feedback, thanks !
I'm now investigating the unstable tests.